apps/mobile/src/navigation/MMBStack.js

```diff
@@ -56,6 +58,18 @@ const StackNavigator: NavigationNavigator<
         title: 'Other',
       },
     },
+    [Routes.SERVICES_FLIGHT_SERVICES]: {
+      screen: FlightServicesScreen,
+      navigationOptions: {
+        title: 'Other',
```

Comment:
Probably should be `Flight Services` and next one `Trip Services`.

Reply:
Good catch!